meadow-client/src/Simulation.purs

```diff
@@ -542,6 +438,15 @@ transactionButtons state =
           , enabled (state.oldContract /= Nothing)
           , onClick $ Just <<< HQ.action <<< const ResetSimulator
           ] [text "Reset"]
+      , button
+          [ classes
+              [ btn
+              , btnPrimary
+              , ClassName "transaction-btn"
+              ]
+          -- , enabled (state.oldContract /= Nothing)
+          , onClick $ Just <<< HQ.action <<< const Undo
+          ] [text "Undo"]
       ]
```

Comment:
We probably want to disable the button if the history is empty (for visual feedback mainly)

Reply:
done